### ISSUE TITLE
Support Multi-gpu Training

### ISSUE THREAD

**Issue by Eaphan** [JonasSchult/Mask3D#185]
Thanks for sharing. If I want to change the code to support multi-gpu training, what's the key point? It seems strange that the performance deteriorate when using multiple gpus in the pytorch-lightning framework.
@JonasSchult @francisengelmann

**Comment by zhao-py** [comment]
> Thanks for sharing. If I want to change the code to support multi-gpu training, what's the key point? It seems strange that the performance deteriorate when using multiple gpus in the pytorch-lightning framework. [@JonasSchult](https://github.com/JonasSchult) [@francisengelmann](https://github.com/francisengelmann)

Sorry to bother you, but have you found an appropriate solution for multi-gpu training?

**Comment by francisengelmann** [comment]
You most likely have to tune the learning rate with increased batch size, maybe loss weights etc - does that help? Right now there is no one actively developing this repository (this work was presented at ICRA'23), although feel free to create a pull request.
